pushers/backends/slack/slack.go

```diff
@@ -72,7 +72,7 @@ func init() {
 
 // Send delivers the giving push messages to the required slack channel.
 // TODO: Ask if Send shouldnt return an error to allow proper delivery validation.
-func (mc MessageChannel) Send(messages []message.PushMessage) {
+func (mc MessageChannel) Send(messages ...message.Event) {
```

Comment:
Yes, we can move the buffering to the channel level and use single message.Events